libs/blog/newsletter/feature-newsletter/src/lib/newsletter-error/newsletter-error.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { NewsletterErrorComponent } from './newsletter-error.component';
+
+describe('NewsletterErrorComponent', () => {
```

Comment:
Consider to write some tests

Reply:
AFAIK we are not going to write unit tests for simple dumb components.